docs/content/config/advanced/mail-sieve.md

```diff
@@ -61,6 +61,23 @@ pipe "external-program";
 
 For more examples or a detailed description of the Sieve language have a look at [the official site](http://sieve.info/examplescripts). Other resources are available on the internet where you can find several [examples](https://support.tigertech.net/sieve#sieve-example-rules-jmp).
 
+## Automatic sorting based on subaddresses
```

Comment:
Just a small nitpick: we usually use uppercase headings, so should would in fact be `## Automatic Sorting Based on Subaddresses` :)